webhook_receiver/settings/__init__.py

```diff
@@ -144,9 +145,23 @@
 CELERY_ALWAYS_EAGER = not bool(CELERY_BROKER_URL)
 CELERY_TASK_ALWAYS_EAGER = not bool(CELERY_BROKER_URL)
 
+default_db = env.db(
+    'DJANGO_DATABASE_URL',
+    default='sqlite:///:memory:'
+)
+
+if not default_db:
+    # env.db() returns an empty dictionary if DJANGO_DATABASE_URL is
+    # set to an empty string. This will break when we want to override
+    # default database configuration parameters for production.
+    raise ImproperlyConfigured(
+        'DJANGO_DATABASE_URL was set to an empty string. Provide a '
+        'valid URL or remove DJANGO_DATABASE_URL to use the '
```

Comment:
I'd put "... or unset DJANGO_DATABASE_URL to use the default."

(What you do if you want to remove an envar from the shell environment is run the `unset` command, hence we should use that, rather than "remove").

Reply:
Makes sense. I'll fix. Thanks :)